packages/api/lib/transformers/classifyTransformer.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Transform = require('stream').Transform,
+  inherits = require('util').inherits,
+  moment = require('moment');
+
+const classifyTransformer = function (classifyTransformerOptions, streamOptions) {
+  if (!(this instanceof classifyTransformer)) {
+    return new classifyTransformer(classifyTransformerOptions, streamOptions);
+  }
+
+  if (!streamOptions) {
+    streamOptions = {};
+  }
+  streamOptions.decodeStrings = false;
+  streamOptions.objectMode = true;
+
+  Transform.call(this, streamOptions);
+};
+
+classifyTransformer.prototype.defineState = function defineState (data, cb) {
```

Comment:
parameter `cb` always called without parameters. Could be removed and called in `_transform` function